packages/terra-graphs/package.json

```diff
@@ -1,6 +1,9 @@
 {
   "name": "@cerner/terra-graphs",
   "main": "lib/Graphs.js",
+  "publishConfig": {
+    "access": "public"
+  },
   "private": true,
```

Comment:
Are we keeping this private until the first release?

Reply:
yeah, this repo is nothing but a placeholder for now.